.goreleaser.yml

```diff
@@ -155,68 +155,56 @@ builds:
       - pivkey
       - pkcs11key
 
-  - id: sget
-    binary: sget-{{ .Os }}-{{ .Arch }}
-    no_unique_dist_dir: true
-    mod_timestamp: '{{ .CommitTimestamp }}'
-    main: ./cmd/sget
-    flags:
-      - -trimpath
-    goos:
-      - linux
-      - darwin
-      - windows
-    goarch:
-      - amd64
-      - arm64
-      - arm
-      - s390x
-      - ppc64le
-      - riscv64
-    goarm:
-      - '7'
-    ignore:
-      - goos: windows
-        goarch: arm64
-      - goos: windows
-        goarch: arm
-      - goos: windows
-        goarch: s390x
-      - goos: windows
-        goarch: ppc64le
-    ldflags:
-      - "{{ .Env.LDFLAGS }}"
-    env:
-      - CGO_ENABLED=0
+- id: linux-cosigned
+  binary: cosigned-linux-{{ .Arch }}
+  no_unique_dist_dir: true
+  main: ./cmd/cosign/webhook
```

Comment:
from where this come from? this was deprecated a long time and not have it own repo (policy-controller)